src/api/app/views/webui/main/_status_messages.html.haml

```diff
@@ -1,3 +1,7 @@
+-# This is needed since inferring the policy class with `policy(StatusMessage.new)` would use StatusMessagePolicy which is for the API
+  Anonymous users can see this view, so we need to set `ensure_logged_in` to `false`
+- status_message_policy = Webui::StatusMessagePolicy.new(User.possibly_nobody, StatusMessage.new, ensure_logged_in: false)
```

Comment:
Without the changes in the policy, `ensure_logged_in` would still be `true`.

See:
```ruby
[4] pry(main)> opts = { ensure_logged_in: false }
=> {:ensure_logged_in=>false}
[5] pry(main)> opts.merge(ensure_logged_in: true)
=> {:ensure_logged_in=>true} # Wrong! It should be false, the default value has precedence over what I'm passing
[6] pry(main)> { ensure_logged_in: true }.merge(opts)
=> {:ensure_logged_in=>false}
```